internal/cloud/cloud.go

```diff
@@ -29,7 +29,7 @@ import (
 	"go.jetpack.io/devbox/internal/ux/stepper"
 )
 
-func Shell(ctx context.Context, w io.Writer, projectDir string, githubUsername string) error {
+func Shell(ctx context.Context, w io.Writer, projectDir string, githubUsername string, interactive bool) error {
```

Comment:
instead of a boolean argument, could we split this function into:
- `cloud.EnsureVMForUser` 
- `cloud.Shell`

That would feel like a more reusable API

Reply:
you mean break this function into 2 different functions?
yeah I think it would make the logic simpler. Let me try it and see how the code turns out.